WooCommerce/WooCommerceTests/ViewRelated/Dashboard/DashboardViewModelTests.swift

```diff
@@ -415,5 +596,17 @@ private extension DashboardViewModelTests {
         }
     }
 
+    /// Returns mocked saved cards or empty.
+    func mockLoadDashboardCards(withStoredCards cards: [DashboardCard]? = nil) {
```

Comment:
Nit: IMO we should remove the default value to make the tests explicitly send `nil` for readability.

Reply:
Great point, I changed it to require passing empty array for no saved cards, for a bit more clarity and make the code a bit shorter

https://github.com/woocommerce/woocommerce-ios/pull/12842/commits/8fd9bb1788b220106f219b11899ecc5eb69e19b2